examples/cis3-nft-sponsored-txs/Cargo.toml

```diff
@@ -16,6 +16,8 @@ concordium-cis2 = {path = "../../concordium-cis2", default-features = false}
 
 [dev-dependencies]
 concordium-smart-contract-testing = { path = "../../contract-testing" }
+ed25519-dalek = "1.0"
+rand = "0.7.0"
```

Comment:
I guess rand is needed, but I'd prefer if we did not require users to manually add the dalek crate. They should be able to just use signature stuff from base. If this is not available we should add it.